backend/mail-service/api/src/main/kotlin/MailDescriptions.kt

```diff
@@ -22,9 +22,33 @@ data class SendBulkRequest(
     val messages: List<SendRequest>
 )
 
+data class SendSupportEmailRequest(
+    val fromEmail: String,
+    val subject: String,
+    val message: String
+)
+
 object MailDescriptions : CallDescriptionContainer("mail") {
     val baseContext = "/api/mail"
 
+    val sendSupport = call<SendSupportEmailRequest, Unit, CommonErrorMessage>("sendSupport") {
```

Comment:
```suggestion
    val sendSupport = call<SendSupportEmailRequest, SendSupportEmailResponse, CommonErrorMessage>("sendSupport") {
```

Also add `typealias SendSupportEmailResponse = Unit`.